docs/tables/snowflake_resource_monitor.md

```diff
@@ -0,0 +1,45 @@
+# Table: snowflake_warehouse
```

Comment:
@paulsewardasc Can you please add the query results (sensitive information removed) for all 3 examples in this doc to the PR description under the `Example query results` section? For instance, please see the `Example query results` section in https://github.com/turbot/steampipe-plugin-aws/pull/984 (you can ignore the integration test results).

This will help match column values with column types/descriptions, and also helps to see real data with these examples. Thanks!

Reply:
When I can get the code to work I will add some results.

Reply:
Examples added